t/test-data/Models/hr-inclNOxExtendedWithFilters/Livestock.nhd

```diff
@@ -174,19 +206,47 @@ gui      = Livestock,Tierhaltung,Production animale,Livestock
   ++formula
     add( 
       scale( 
-        Sum(tan_excretion, Livestock::OtherCattle::Excretion) P+ 
-        Sum(tan_excretion, Livestock::DairyCow::Excretion), 
+        Sum(n_excretion, Livestock::Equides) P+ 
+        Sum(n_excretion, Livestock::SmallRuminants) P+ 
+        Sum(n_excretion, Livestock::RoughageConsuming), 
         0 ),
-      1 ) P+
-    scale( 
-      Sum(tan_excretion, Livestock::Pig::Excretion) P+
-      Sum(tan_excretion, Livestock::FatteningPigs::Excretion) P+
-      Sum(tan_excretion, Livestock::Equides::Excretion) P+
-      Sum(tan_excretion, Livestock::SmallRuminants::Excretion) P+
-      Sum(tan_excretion, Livestock::RoughageConsuming::Excretion) P+
-      Sum(tan_excretion, Livestock::Poultry::Excretion), 
-      0 );
-    
+      1 );
+
++test
+  print = check
+  ++formula
+    hashsign( Sum(n_from_poultry_layers_growers_other_poultry,Livestock::Poultry) );
+
+# +is_layers_growers_other_poultry
```

Comment:
Don't add new code that is commented out. I guess this is not really related to the purpose of this PR.

Reply:
Cleaned